osbuild.spec

```diff
@@ -0,0 +1,57 @@
+%global         pypi_name osbuild
+
+Name:           osbuild
+Version:        1
+Release:        1%{?Dist}
+License:        ASL 2.0
+
+URL:            https://github.com/larskarlitski/osbuild
+
+Source0:        %{name}-%{version}.tar.gz
+BuildArch:      noarch
+Summary:        A build system for OS images
+
+BuildRequires:  python3-devel
+
+Requires: bash
+Requires: coreutils
+Requires: dnf
+Requires: e2fsprogs
+Requires: glibc
+Requires: policycoreutils
+Requires: qemu-img
+Requires: systemd
+Requires: systemd-container
+Requires: tar
+Requires: util-linux
+
+%{?python_enable_dependency_generator}
+
+%description
+A build system for OS images
+
+%{?python_provide:%python_provide python3-%{pypi_name}}
```

Comment:
I am not sure with this line, because I used simply `osbuild` as the package name instead of `python3-osbuild`.

Taken from the docs:

> 

%{python_provide} | (Lua script) | Given a package name, evaluates to either Provides: python-$name or nothing at all depending on the Pythonversion. See The %python_provide macro for an example.
-- | -- | --


Source: https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/

Reply:
Good question: It says that libraries must be named `python3-{name}`, but applications don't. We have both with the same name. I hope we don't need to split it?

Reply:
I guess we have to. Because now we also provide `osbuild` as a python module.